### PR TITLE
Added less_equal function and test to Paddle frontend

### LINKED ISSUE #15815: less_equal

### PR DESCRIPTION
**Pull request by whatzittoya**
Close #15815

### CODE REVIEW

**Comment by KareemMAX** [comment]
add_frontend_checklist

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_math.py [hunk 1]
```diff
@@ -190,3 +190,32 @@ def test_paddle_asin(
...
+@handle_frontend_test(
+    fn_tree="paddle.tensor.math.less_equal",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric"),
+        num_arrays=2,
+        shared_dtype=True,
+    ),
+)
+
+def test_tensorflow_less_equal(
```
Comment (KareemMAX) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_math.py:204]: Naming is not correct here 
```suggestion
def test_paddle_less_equal(
```

**Review by KareemMAX — CHANGES_REQUESTED** [review]
Hey @whatzittoya, awesome work 🚀

Sadly some check have been missing as you can see on the checklist I placed. Can you please address them?

Please note that you haven't passed any of the tests, also you didn't use `pre-commit` on linting

**Review by KareemMAX — CHANGES_REQUESTED** [review]
Hey @whatzittoya,
Seems like I wasn't paying attention, there is no function called `paddle.less_equal` thus it's failing all the tests. And it seems like you were implementing a method for `paddle.Tensor`. That means you didn't have the function nor the test in their right files. https://github.com/unifyai/ivy/pull/16195 might be a good place where you can see how are implementing it.

Sadly I'll be resetting the checklist

**Comment by KareemMAX** [comment]
add_frontend_checklist

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Pull request closed without merging**